hack/ci/e2e-conformance/calico.yaml

```diff
@@ -0,0 +1,746 @@
+                readOnly: false
+              - mountPath: /var/run/calico
+                name: var-run-calico
+                readOnly: false
+              - mountPath: /var/lib/calico
+                name: var-lib-calico
+                readOnly: false
+            volumes:
+            # Used by calico-node.
+            - name: lib-modules
+              hostPath:
+                  path: /lib/modules
+            - name: var-run-calico
+              hostPath:
+                  path: /var/run/calico
+            - name: var-lib-calico
+              hostPath:
+                  path: /var/lib/calico
+            - name: xtables-lock
+              hostPath:
+                  path: /run/xtables.lock
+                  type: FileOrCreate
+            # Used to install CNI.
+            - name: cni-bin-dir
+              hostPath:
+                  path: /opt/cni/bin
+            - name: cni-net-dir
+              hostPath:
+                  path: /etc/cni/net.d
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: calico-node
+    namespace: kube-system
+
+---
+# Source: calico/templates/calico-kube-controllers.yaml
+
+# See https://github.com/projectcalico/kube-controllers
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: calico-kube-controllers
+    namespace: kube-system
+    labels:
+        k8s-app: calico-kube-controllers
+spec:
+    # The controllers can only have a single active instance.
+    replicas: 1
+    selector:
+        matchLabels:
+            k8s-app: calico-kube-controllers
+    strategy:
+        type: Recreate
+    template:
+        metadata:
+            name: calico-kube-controllers
+            namespace: kube-system
+            labels:
+                k8s-app: calico-kube-controllers
+            annotations:
+                scheduler.alpha.kubernetes.io/critical-pod: ''
+        spec:
+            nodeSelector:
+                kubernetes.io/os: linux
+            tolerations:
+            # Mark the pod as a critical add-on for rescheduling.
+            - key: CriticalAddonsOnly
+              operator: Exists
+            - key: node-role.kubernetes.io/master
+              effect: NoSchedule
+            serviceAccountName: calico-kube-controllers
+            priorityClassName: system-cluster-critical
+            containers:
+            - name: calico-kube-controllers
+              image: calico/kube-controllers:v3.13.1
+              env:
+              # Choose which controllers to run.
+              - name: ENABLED_CONTROLLERS
+                value: node
+              - name: DATASTORE_TYPE
+                value: kubernetes
+              readinessProbe:
+                  exec:
+                      command:
+                      - /usr/bin/check-status
+                      - -r
+
+---
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: calico-kube-controllers
+    namespace: kube-system
+---
+# Source: calico/templates/calico-etcd-secrets.yaml
```

Comment:
not sure do we need down those files??

Reply:
I added the file because I removed the flex volume driver to see if this stabilizes the test. If we have to deploy calico without flex driver I would prefer the stable file here instead of trying to patch the latest calico yaml automatically (which will probably break with a new Calico release)